### PR TITLE
Adds a new gene as the Preternis inert mutation, Ravenous

### PR DESCRIPTION
**Pull request by Moltijoe**
# Why is this good for the game?
Adds more depth to genetics for species
also lol hungy

makes the host hungrier, eat faster, and have a bottomless stomach

:cl:  
rscadd: Adds the Ravenous gene as the preternis inert mutation
/:cl:

### CODE REVIEW

File: code/datums/mutations/ravenous.dm [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	text_gain_indication = span_notice("You need more and more and MORE!")
+	text_lose_indication = span_notice("That should be enough for now.")
+	instability = 10 //could be an upside maybe, or could force it on some poor sap while stabilized
+
+/datum/mutation/human/ravenous/on_acquiring(mob/living/carbon/human/owner)
+	if(..())
+		return
+	ADD_TRAIT(owner, TRAIT_EAT_MORE, "ravenous")
+	ADD_TRAIT(owner, TRAIT_BOTTOMLESS_STOMACH, "ravenous")
+	ADD_TRAIT(owner, TRAIT_VORACIOUS, "ravenous")
```
Comment (iamthesilly) [inline code/datums/mutations/ravenous.dm:16]: Shouldn't you be using GENETIC_MUTATION define for the source?

**Review by SapphicOverload — CHANGES_REQUESTED** [review]

File: code/datums/mutations/ravenous.dm (hunk 1 above)
Comment (Moltijoe) [reply code/datums/mutations/ravenous.dm:16]: i was following what space adaptation was doing, which was a standard string

**Review by SapphicOverload — APPROVED** [review]
works on my machine

**Review by MajManatee — APPROVED** [review]
a silly, non-powergamey mutation. wonderful.

**Pull request merged**